src/main/java/io/bittiger/ads/activity/AdsSelect.java

```diff
@@ -0,0 +1,65 @@
+package io.bittiger.ads.activity;
+
+import io.bittiger.ads.model.Ad;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.HashSet;
+import java.lang.String;
+
+public class AdsSelect {

```

Comment:
Class's name should be object, not an action. So in this case make it `AdsSelection`